src/main/java/it/gov/pagopa/pu/organization/entity/BrokerEntity.java

```diff
@@ -0,0 +1,47 @@
+package it.gov.pagopa.pu.organization.entity;
```

Comment:
let's use more generic name `model`
```suggestion
package it.gov.pagopa.pu.organization.model;
```